components/autocomplete/index.js

```diff
@@ -242,6 +250,10 @@ export class Autocomplete extends Component {
 		const { open, range, query } = this.state;
 		const { getOptionCompletion } = open || {};
 
+		if ( option.isDisabled ) {
```

Comment:
I wondered why this was necessary due to the button actually being disabled. Now I see it is due to the fact that our keyboard listener calls `select` with indiscriminately when ENTER is pressed. I wonder if it would be clearer to conditionally call `select` in `handleKeyDown`. Just sharing my experience in case you concur.

Reply:
Yeah, we re-use `select()` in both the `keyDown` and `onClick` handlers. I think it's OK to leave as is.